core/services/address/CountrySubRegionDao.php

```diff
@@ -228,4 +236,37 @@ private function saveSubRegionData(stdClass $country, $sub_regions = array())
         }
         return $results;
     }
+
+
+    /**
+     * @param string $CNT_ISO
+     * @since $VID:$
+     * @return array
+     * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     * @throws ReflectionException
+     */
+    private function getExistingStateAbbreviations($CNT_ISO)
+    {
+        $existing_sub_region_IDs = [];
+        $existing_sub_regions = $this->state_model->get_all(array(
+            array(
+                'Country.CNT_ISO' => array(
+                    'IN',
+                    [$CNT_ISO]
```

Comment:
Oh nice, I forgot that short form array syntax was introduced with PHP 5.4!

Reply:
Even though I approved this, there's some failing code linting issues that need to be addressed.